src/kokab/n_pls_m_gs/sage.py

```
@@ -202,23 +206,64 @@
                     ("chi1_variance_pl", N_pl),
                     ("chi2_mean_g", N_g),
                     ("chi2_mean_pl", N_pl),
-                    ("chi2_scale_g", N_g),
-                    ("chi2_scale_pl", N_pl),
                     ("chi2_variance_g", N_g),
                     ("chi2_variance_pl", N_pl),
                 ]
             )
 
+            def mean_variance_check(**kwargs) -> Array:
+                check = lambda m, v: jnp.less_equal(v, m * (1 - m))
```

Pylint (3.3.7)

[convention] Lambda expression assigned to a variable. Define a function using the "def" keyword instead.

(C3001)